src/PKSim.CLI.Core/MinimalImplementations/CLIDataImporter.cs

```diff
@@ -12,9 +14,15 @@ public IEnumerable<DataRepository> ImportDataSets(IReadOnlyList<MetaDataCategory
          return Enumerable.Empty<DataRepository>();
       }
 
-      public DataRepository ImportDataSet(IReadOnlyList<MetaDataCategory> metaDataCategories, IReadOnlyList<ColumnInfo> columnInfos, DataImporterSettings dataImporterSettings)
+      public IEnumerable<DataRepository> ImportFromConfiguration(ImporterConfiguration configuration, bool promptForConfirmation, IReadOnlyList<MetaDataCategory> metaDataCategories,
+         IReadOnlyList<ColumnInfo> columnInfos, DataImporterSettings dataImporterSettings)
       {
-         return new NullDataRepository();
+         return Enumerable.Empty<DataRepository>();
+      }
+
+      (IEnumerable<DataRepository> DataRepositories, ImporterConfiguration Configuration) IDataImporter.ImportDataSets(IReadOnlyList<MetaDataCategory> metaDataCategories, IReadOnlyList<ColumnInfo> columnInfos, DataImporterSettings dataImporterSettings)
+      {
+         return (new List<DataRepository>(), null);
```

Comment:
use return Enumerable.Empty<DataRepository>() as in the line above for consistency

Reply:
Done